bitnami/discourse/templates/service.yaml

```
@@ -34,6 +34,6 @@ spec:
       nodePort: null
       {{- end }}
     {{- if .Values.service.extraPorts }}
-    {{- include "common.tplvalues.render" (dict "value" .Values.service.extraPorts "context" $) | indent 4 }}
+    {{- toYaml .Values.ingress.extraPorts | nindent 4 }}
```

```suggestion
   {{- include "common.tplvalues.render" (dict "value" .Values.service.extraPorts "context" $) | nindent 4 }}
```

This change should be enought

---

Is this solution just to switch indent with nindent?

---

UPdated with bumped chart version

---

Yes changing indent by nindent  works